stubs/psutil/psutil/__init__.pyi

```diff
@@ -150,7 +154,7 @@ class Process:
         def memory_maps(self, grouped: bool = ...): ...
     def open_files(self) -> list[popenfile]: ...
     def connections(self, kind: str = ...) -> list[pconn]: ...
-    def send_signal(self, sig) -> None: ...
+    def send_signal(self, sig: int | signal.Signals) -> None: ...
```

Comment:
`int` would be enough:
```suggestion
    def send_signal(self, sig: int) -> None: ...
```

This is because `Signals` is an `IntEnum`:

```python
>>> from signal import Signals
>>> isinstance(Signals(9), int)
True
>>> Signals.__bases__
(<enum 'IntEnum'>,)
```

Reply:
Done! 👍